src/naming-strategy/NamingStrategyInterface.ts

```diff
@@ -162,11 +162,6 @@ export interface NamingStrategyInterface {
      */
     prefixTableName(prefix: string, tableName: string): string
 
-    /**
-     * Gets the name of the alias used for relation joins.
-     */
-    eagerJoinRelationAlias(alias: string, propertyPath: string): string
```

Comment:
This function was not being used at all, and just brings noise when trying to extend the interface